pkg/display/allcoin/allCoin_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright © 2021 Bhargav SNV bhargavsnv100@gmail.com
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package allcoin
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFilterRows(t *testing.T) {
+	tests := []struct {
+		allRows      [][]string
+		filter       string
+		filteredRows [][]string
+	}{
+		{
+			allRows: [][]string{
+				{"", "BTC", "", "", "", "bitcoin"},
+				{"", "ETH", "", "", "", "ethereum"},
+				{"", "LTC", "", "", "", "litecoin"},
+				{"", "BTC", "", "", "", "bitcoin"},
+				{"", "BTC", "", "", "", "bitcoin"},
+			},
```

Comment:
@nolancon maybe add a few cases where filter happens using the name of the coin? As of now the `filterRows()` function checks if the given filter string exists in either the symbol name or the name itself.